2.4/run-mongod.sh

```diff
@@ -6,6 +6,8 @@ source /var/lib/mongodb/.bashrc
 
 set -eu
 
+export HOME=/var/lib/mongodb
```

Comment:
So this would be right approach, unfortunately there is a bug in mongodb24 scl and the pkg is setting wrong HOME dir for the mongodb user. Check https://bugzilla.redhat.com/show_bug.cgi?id=1206151 and I doubt they will fix it any time soon.
So Im wondering if this change couldnt break us somehow.